packages/block-library/src/latest-comments/style.scss

```diff
@@ -1,3 +1,7 @@
+.wp-block-latest-comments {
```

Comment:
can't we just use `ol.wp-block-latest-comments` here and avoid any specific editor style?

Reply:
I did not understand. Did you mean to add this comment to the editor.scss file?
That might be possible now after https://github.com/WordPress/gutenberg/pull/30034.
I will test it.

Reply:
no I mean replace the selector above with `ol.wp-block-latest-comments` and avoid any style in "editor.scss". It worked in my testing.

Reply:
The `ol.wp-block-latest-comments` is still overridden by
```
.editor-styles-wrapper ol, .editor-styles-wrapper ul {
    margin: revert;
    padding: revert;
}
```
that is why `ol.wp-block-latest-comments.wp-block-latest-comments` was used in the editor style.

Reply:
It is only overridden if the block is part of the post when it gets loaded in the editor. this is an issue with the lazy loading behavior of block assets in the editor. It affects more blocks than just the comments block I believe we should fix the root issue (stylesheet loading order) not its consequences. I already talked with @aristath and @gziolo about it.